src/main/webapp/js/instructor.es6

```diff
@@ -369,6 +369,25 @@ function attachEventToDeleteStudentLink() {
     });
 }
 
+function remindButtonAjax(urlLink) {
+    const $statusMessage = $('#statusMessagesToUser');
+    $.ajax({
+        type: 'POST',
+        url: urlLink,
+        beforeSend() {
+            $statusMessage.html("<img src='/images/ajax-loader.gif'/>");
```

Comment:
Missed this - this should use `.html('<img src="/images/ajax-loader.gif"/>')` instead, although I am aware that both the single quote and double quote variants are present in the code.